meal-sharing/src/backend/api/meals.js

```diff
@@ -0,0 +1,136 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+router.get("/", async (request, response) => {
+  try {
+    let meal = await knex("meal");
```

Comment:
While it works as expected there a few things that could be done better and if you're up for it you can improve it:
- The route will not work with multiple parameters at the same time as expected. If for instance I pass in both 'maxPrice' and 'availableReservations' you will return meals filtered only by the MaxPrice parameters since that is the one you check first.
- Check one more comment below for the isNaN check

I strongly suggest you watch the following [video ](https://www.youtube.com/watch?v=4KV79W4g2QU&t=418s)made by Orhan at HYF which explains how you can achieve the optimal behavior with the multiple params.